### PR TITLE
Install flake8 and add new workflow

### LINKED ISSUE #15: Install flake8
Install pipenv flake8
Add new workflow for CI

### PR DESCRIPTION
**Pull request by ghost**
fix #15 
- Install flake8 
- Add flake8.yml file with new workflow
- Fixing errors in files (missing blank line, remove unused imports)

### CODE REVIEW

**Review by VeraTs — COMMENTED** [review]
Looks ok but I think it should be 1 commit

File: .github/workflows/flake8.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        container: fedora:32
+        steps:
+            - name: Install Pipenv and Git
+                run: dnf install -y pipenv git
+            - name: Checkout code
+                uses: actions/checkout@v2
+            - name: Setup environment
+                run: pipenv sync --dev
+            - name: Run Flake8
+                run: pipenv run flake8 --max-line-length 120
```
Comment (lmilbaum) [inline .github/workflows/flake8.yml]: missing new line

File: homepage/admin.py [hunk 2]
```diff
@@ -1 +1 @@
-from django.contrib import admin
+# from django.contrib import admin
```
Comment (lmilbaum) [inline homepage/admin.py]: Remove those comments

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: homepage/admin.py (hunk 2 above)
Comment (ghost) [reply homepage/admin.py]: isn't it necessary? 
'django.contrib' is not use. 
if I will remove the comment, the test will fail.

Comment (lmilbaum) [reply homepage/admin.py]: Delete the whole line. What is the purpose of the comment at all?

**Review by VeraTs — APPROVED** [review]
LGTM

File: .github/workflows/flake8.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
```
Comment (lmilbaum) [inline .github/workflows/flake8.yml]: What is going on with this error?

**Comment by lmilbaum** [comment]
I would expect to see the flake8 workflow used on this PR.

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: .github/workflows/flake8.yml (hunk 3 above)
Comment (ghost) [reply .github/workflows/flake8.yml]: Which error do you mean? 
I created the .yml file exactly like in the intro pdf.

**Review by lmilbaum — APPROVED** [review]

**Pull request merged**